docs/PAYJOIN.md

```diff
@@ -115,16 +138,79 @@ that if you don't bother to do anything special, the coins in those two mixdepth
 (The new standard (BIP49) *should* be compatible with TREZOR, Ledger, Electrum, Samourai and some others,
 including the 12 word seed, although consider privacy concerns when sending addresses to remote servers!).
 
+<a name="doing" />
+
 ### Doing a PayJoin payment.
 
+<a name="bip78" />
+
+#### Using BIP78 payjoins to pay a merchant.
+
+The process here is to use the syntax of sendpayment.py:
+
+```
+(jmvenv)a$ python sendpayment.py -m 0 walletname.jmdat "bitcoin:bc1qxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx?amount=0.05933201&pj=https://some/url/payjoin"
+```
+
+Notes on this:
+* Payjoins BIP78 style are done using the `sendpayment` script (there is no Qt support yet, but it will come later).
+* They are done using BIP21 uris. These can be copy/pasted from a website (e.g. a btcpayserver invoice page), note that double quotes are required because the string contains special characters. Note also that you must see `pj=` in the URI, otherwise payjoin is not supported by that server.
```

Comment:
```suggestion
* They are done using BIP21 URIs. These can be copy/pasted from a website (e.g. a btcpayserver invoice page), note that double quotes are required because the string contains special characters. Note also that you must see `pj=` in the URI, otherwise payjoin is not supported by that server.
```

Reply:
Done in https://github.com/JoinMarket-Org/joinmarket-clientserver/pull/633/commits/b5e87ba3052cf9bc85e016bfd582b70d7066eb98